core/src/main/java/com/instaclustr/esop/impl/AbstractOperationRequest.java

```diff
@@ -48,8 +48,7 @@ public abstract class AbstractOperationRequest extends OperationRequest {
     public RetrySpec retry = new RetrySpec();
 
     @Option(names = {"--cc", "--concurrent-connections"},
```

Comment:
I think we should add "--parallelism"  name here, do not remove already existing because if people use it already we would break it for them.